site/app/models/gradeable/TaGradedGradeable.php

```diff
@@ -22,8 +22,8 @@ class TaGradedGradeable extends AbstractModel {
     private $graded_gradeable = null;
     /** @prop @var int The id of this gradeable data */
     protected $id = 0;
-    /** @prop @var string The grader's overall comment */
-    protected $overall_comment = "";
+    /** @prop @var string[] indexed by user_id. Overall comment made by each grader. */
+    protected $overall_comment = [];
```

Comment:
Let's change this to `overall_comments` if it's an array and not a string.

Reply:
Updated